Tests/t_ldapobject.py

```diff
@@ -421,6 +428,33 @@ def test_multiple_starttls(self):
             l.simple_bind_s(self.server.root_dn, self.server.root_pw)
             self.assertEqual(l.whoami_s(), 'dn:' + self.server.root_dn)
 
+    @requires_tls()
+    @unittest.skipUnless(
+        hasattr(ldap, "OPT_X_TLS_PEERCERT"),
```

Comment:
Sounds like you still need to register this in `Lib/ldap/constants.py` to get the tests enabled?